bootq/src/main/java/life/genny/bootq/models/BatchLoading.java

```diff
@@ -93,6 +105,96 @@ public void persistProject(RealmUnit rx) {
         persistQuestionQuestions(rx.getQuestionQuestions(), rx.getCode());
     }
 
+    public void loadToSqlite(RealmUnit rx) throws SQLException, IOException {
+        Set<String> moduleUnitNames = new HashSet<>();
+        int i = 1;
+        for (ModuleUnit moduleUnit : rx.getModule().getDataUnits()) {
+            String moduleUnitName = moduleUnit.getModuleName();
+            if(moduleUnitNames.contains(moduleUnitName)) {
+                moduleUnitName = moduleUnitName + i++;
+            }
+            moduleUnitNames.add(moduleUnitName);
+            log.infof("Loading into module: %s - %s", moduleUnit.getName(), moduleUnitName);
```

Comment:
is this a duplication handler that allows duplicates through, ensuring they're marked as duplicate? In this case should we update the module name as such or leave it be?

Reply:
The duplication is allowed in case the module names are common across multiple sheets. This is temporary only. It could be removed after careful consideration.

Reply:
Setting the updated name to moduleUnit.